lib/rubocop-veeqo.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module RubocopVeeqo
-  VERSION = '0.0.5'
+  VERSION = '0.0.6-temp'
```

Comment:
Why temp?

Reply:
good catch, I was releasing `pre` version and forgot to change it